### ISSUE TITLE
Rewrite spam-checker.js async function to parse content after checking for empty title

### ISSUE THREAD

**Issue by c3ho** [Seneca-CDOT/telescope#250]
Currently for #193 part of the async function contains the following code

```
  // The main body of the blog post with all HTML tags removed
  const noTagsDesc = await textParser.run(content.description);

  // If the title is empty, post is considered spam
  if (content.title != null) {
    if (content.title !== ''
        && !filter.isSpam(content.title)) {
      // If the word count is under MINWORDS, all characters are capital,
      // or spam-filter returns true, post is considered spam
      if (getWordCount(noTagsDesc) > MINWORDS
          && getCapitalLetters(noTagsDesc) < getAllLetters(noTagsDesc)
          && !filter.isSpam(noTagsDesc)) {
        return false;
      }
      return true;
    }
    return true;
  }
  return true;
```
We should rewrite it so ` const noTagsDesc = await textParser.run(content.description);` is called after we do all the checking for content.title.

>  we should check for title spam before calling text-parser since content.title doesn't rely on the text-parser and we can save an operation there. No point in parsing the whole description of the body only to have the content.title empty.

**Comment by jordanhui19** [comment]
I can work on this, since I made the spam checker

**Comment by c3ho** [comment]
Fixed via #329
